hyperspy/io.py

```diff
@@ -771,6 +772,14 @@ def save(filename, signal, overwrite=None, **kwds):
             f"Please try one of {strlist2enumeration(yes_we_can)}"
         )
 
+    if writer.non_uniform_axis is False and nua is False:
+        yes_we_can = [plugin.format_name for plugin in io_plugins
```

Comment:
While you are at it: could you rename `yes_we_can` to `compatible_writers` or something similar, it mislead me during the review!

Reply:
Sure, though I found it funny ;-)

Reply:
Indeed!